EIPS/eip-5792.md

```diff
@@ -256,9 +256,9 @@ type AtomicBatchCapability = {
 };
 ```
 
-The only valid JSON-RPC values for this capability are `true` or `false`; if a returned value for "supported" value is typed as a string or number, it SHOULD be treated as malformed.
+The only valid JSON-RPC values for this capability's only member, `supported`, are `true` or `false`; if a returned value for `supported` is typed as a string or number, it SHOULD be considered malformed.
 
-For each chain on which a wallet can submit multiple calls atomically, the wallet SHOULD include an `atomicBatch` capability with a `supported` field equal to `true`. For each chain where it cannot, it SHOULD return the same field as `false`. Unspecified support for a given, named chain SHOULD be interpreted as not exposing this information to the application, and as such, applications SHOULD NOT interpret an absent value as equivalent to `false`.
+For each chain on which a wallet can submit multiple calls atomically, the wallet SHOULD include an `atomicBatch` capability with a `supported` field equal to `true` to authorize the caller to use this capability. For each chain where it cannot, it MAY return the same field as `false`, or MAY return no capability object at all. An application SHOULD NOT interpret any `atomicBatch` object as an authorization without a `supported` of `true`.
```

Comment:
i still disagree with the wording here, capability expression is separate from authorization. let's just leave this as it was originally?

Reply:
ok